internal/host/host.go

```diff
@@ -967,11 +974,11 @@ func (m *Manager) reportValidationStatusChanged(ctx context.Context, vc *validat
 					} else if vc.infraEnv != nil {
 						m.metricApi.HostValidationChanged(vc.infraEnv.OpenshiftVersion, vc.infraEnv.EmailDomain, models.HostValidationID(v.ID))
 					}
-					eventgen.SendHostValidationFallingEvent(ctx, m.eventsHandler, *h.ClusterID, *h.ID,
+					eventgen.SendHostValidationFallingEvent(ctx, m.eventsHandler, getClusterIDForEvent(h), *h.ID,
```

Comment:
this issue will be handled by MGMT-7806

Reply:
Is it going to be merged ? Otherwise the service crashes and it cannot be merged